GeneralsMD/Code/Libraries/Source/WWVegas/WWLib/rawfile.cpp

```diff
@@ -51,22 +51,17 @@
  *   RawFileClass::Write -- Writes the specified data to the buffer specified.                 *
  * - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - */
 
-
+// TheSuperHackers @compile feliwir 11/04/2025 We removed Win32 API here and use plain C
```

Comment:
Minor: I suggest to not use "I", "We" in comments, to keep it unpersonalized.
Better: "Removes Win32 API and uses plain C for file io."